server/.env.sample

```diff
@@ -65,3 +65,8 @@ EMAIL_PORT=
 EMAIL_SENDER=donotreply@tombolo.com
 
 REALBI_URL=
+
+# For string encryption and decryption. Refer - https://www.geeksforgeeks.org/node-js-crypto-createdecipheriv-method/
+ALGORITHM = 
```

Comment:
Please remind me the idea behind these new properties. Are these used for Cluster credentials? If so, I beleive we are already storing cluster credentials using bcrypt algorithm

Reply:
We are using the same algorithm. The reason behind these change  is because node has deprecated crypto.createDecipher and crypto.createCipher  and introduced crypto.createCipheriv and crypto.createDecipher . 
 We had the algorithm variable inside the file, I moved it out to env file. 
The IV length and Ecryption key are needed for these new functions. The existing cluster_cred secret does not meet the key specification

Reply:
Ok. Will this work with existing cluster credentials? It may not be a big deal as we do not have many people using it. Can we get rid of the cluster_cred property from .env.sample and update the readme accordingly?

Reply:
The existing cluster cred WON'T WORK. The hash saved in the DB is created with different. We need to ask user to remove and add clusters again. The cluster_cred_secret is still used by Github flow. Once we replace there we should be good to get rid of it from .env file.